.github/workflows/versions.yml

```diff
@@ -51,6 +51,26 @@ jobs:
           __tests__/verify-node.sh "${BASH_REMATCH[1]}"
         shell: bash
 
+  v8-canary-syntax:
```

Comment:
Could you please fix yaml file because checks didn't run https://github.com/actions/setup-node/actions/runs/3540135983